### PR TITLE
[FLINK-32849][runtime][JUnit5 Migration] The resourcemanager package of flink-runtime module

### PR DESCRIPTION
**Pull request by RocMarshal**
## What is the purpose of the change

[runtime][JUnit5 Migration] The resourcemanager package of flink-runtime module

## Brief change log

[runtime][JUnit5 Migration] The resourcemanager package of flink-runtime module

## Verifying this change

This change is already covered by existing tests, such as *(please describe tests)*.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (**not applicable** / docs / JavaDocs / not documented)

### CODE REVIEW

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerJobMasterTest.java [hunk 1]
```diff
@@ -207,17 +200,16 @@ public void testRegisterJobMasterFromInvalidAddress() throws Exception {
                         invalidAddress,
                         jobId,
                         TIMEOUT);
-        assertTrue(
-                invalidAddressFuture.get(5, TimeUnit.SECONDS)
-                        instanceof RegistrationResponse.Failure);
+        assertThat(invalidAddressFuture.get(5, TimeUnit.SECONDS))
+                .isInstanceOf(RegistrationResponse.Failure.class);
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerJobMasterTest.java:204]:
```suggestion
        assertThatFuture(invalidAddressFuture)
                .failsWithin(5, TimeUnit.SECONDS)
                .withThrowableOfType(RegistrationResponse.Failure.class)
```

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerJobMasterTest.java [hunk 2]
```diff
@@ -191,12 +184,12 @@ public void testRegisterJobMasterWithUnmatchedLeaderSessionId2() throws Exceptio
                         jobMasterGateway.getAddress(),
                         jobId,
                         TIMEOUT);
-        assertTrue(unMatchedLeaderFuture.get() instanceof RegistrationResponse.Failure);
+        assertThat(unMatchedLeaderFuture.get()).isInstanceOf(RegistrationResponse.Failure.class);
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerJobMasterTest.java:187]:
```suggestion
        assertThatFuture(unMatchedLeaderFuture)
                .withThrowableOfType(RegistrationResponse.Failure.class)
```

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerJobMasterTest.java [hunk 3]
```diff
@@ -169,18 +166,14 @@ public void testRegisterJobMasterWithUnmatchedLeaderSessionId1() throws Exceptio
...
-
-        try {
-            unMatchedLeaderFuture.get(5L, TimeUnit.SECONDS);
-            fail("Should fail because we are using the wrong fencing token.");
-        } catch (ExecutionException e) {
-            assertTrue(ExceptionUtils.stripExecutionException(e) instanceof FencingTokenException);
-        }
+        assertThatThrownBy(() -> unMatchedLeaderFuture.get(5L, TimeUnit.SECONDS))
+                .withFailMessage("Should fail because we are using the wrong fencing token.")
+                .hasCauseInstanceOf(FencingTokenException.class);
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerJobMasterTest.java:171]: How about using `assertThatFuture(invalidAddressFuture)`?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/DefaultJobLeaderIdServiceTest.java [hunk 4]
```diff
@@ -122,23 +117,19 @@ public void testRemovingJob() throws Exception {
...
-            leaderIdFuture.get();
+        assertThat(jobLeaderIdService.containsJob(jobId)).isFalse();
 
-            fail("The leader id future should be completed exceptionally.");
-        } catch (ExecutionException ignored) {
-            // expected exception
-        }
+        assertThatThrownBy(leaderIdFuture::get)
+                .withFailMessage("The leader id future should be completed exceptionally.")
+                .isInstanceOf(ExecutionException.class);
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/DefaultJobLeaderIdServiceTest.java:124]: How about `assertThatFuture`?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerJobMasterTest.java [hunk 5]
```diff
@@ -229,13 +221,12 @@ public void testRegisterJobMasterWithFailureLeaderListener() throws Exception {
...
-        } catch (ExecutionException e) {
-            assertTrue(
-                    ExceptionUtils.stripExecutionException(e) instanceof ResourceManagerException);
-        }
+        assertThatThrownBy(
+                        () ->
+                                registrationFuture.get(
+                                        TIMEOUT.toMilliseconds(), TimeUnit.MILLISECONDS))
+                .withFailMessage("Expected to fail with a ResourceManagerException.")
+                .isInstanceOf(ResourceManagerException.class);
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerJobMasterTest.java:229]: How about assertThatFuture?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerPartitionLifecycleTest.java [hunk 6]
```diff
@@ -195,7 +192,7 @@ public static void registerTaskExecutor(
                 resourceManagerGateway.registerTaskExecutor(
                         taskExecutorRegistration, TestingUtils.TIMEOUT);
 
-        assertThat(registrationFuture.get(), instanceOf(RegistrationResponse.Success.class));
+        assertThat(registrationFuture.get()).isInstanceOf(RegistrationResponse.Success.class);
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerPartitionLifecycleTest.java:195]: How about assertThatFuture?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerServiceImplTest.java [hunk 7]
```diff
@@ -156,12 +151,12 @@ public void grantLeadership_startRmAndConfirmLeaderSession() throws Exception {
                 leaderElection.isLeader(leaderSessionId);
 
         // should start new RM and confirm leader session
-        assertThat(startRmFuture.get(), is(leaderSessionId));
-        assertThat(confirmedLeaderInformation.get().getLeaderSessionID(), is(leaderSessionId));
+        assertThat(startRmFuture.get()).isSameAs(leaderSessionId);
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerServiceImplTest.java:154]: How about assertThatFuture?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerServiceImplTest.java [hunk 8]
```diff
@@ -213,14 +208,14 @@ public void grantLeadership_withExistingLeader_stopExistLeader() throws Exceptio
                 leaderElection.isLeader(leaderSessionId2);
 
         // should terminate first RM, start a new RM and confirm leader session
-        assertThat(terminateRmFuture.get(), is(leaderSessionId1));
-        assertThat(startRmFuture2.get(), is(leaderSessionId2));
-        assertThat(confirmedLeaderInformation.get().getLeaderSessionID(), is(leaderSessionId2));
+        assertThat(terminateRmFuture.get()).isSameAs(leaderSessionId1);
+        assertThat(startRmFuture2.get()).isSameAs(leaderSessionId2);
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerServiceImplTest.java:212]: How about assertThatFuture?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerServiceImplTest.java [hunk 9]
```diff
@@ -254,12 +249,13 @@ public void grantLeadership_withExistingLeader_waitTerminationOfExistingLeader()
         finishRmTerminationFuture.complete(null);
 
         // should start new RM and confirm leader session
-        assertThat(startRmFuture2.get(), is(leaderSessionId2));
-        assertThat(confirmedLeaderInformation.get().getLeaderSessionID(), is(leaderSessionId2));
+        assertThat(startRmFuture2.get()).isSameAs(leaderSessionId2);
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerServiceImplTest.java:252]: How about assertThatFuture?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerServiceImplTest.java [hunk 10]
```diff
@@ -309,12 +305,11 @@ public void revokeLeadership_stopExistLeader() throws Exception {
         leaderElection.notLeader();
 
         // should terminate RM
-        assertThat(terminateRmFuture.get(), is(leaderSessionId));
+        assertThat(terminateRmFuture.get()).isSameAs(leaderSessionId);
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerServiceImplTest.java:308]: How about assertThatFuture?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerServiceImplTest.java [hunk 11]
```diff
@@ -365,7 +360,7 @@ public void nonLeaderRmTerminated_doseNotTerminateService() throws Exception {
 
         // revoke leadership
         leaderElection.notLeader();
-        assertThat(terminateRmFuture.get(), is(leaderSessionId));
+        assertThat(terminateRmFuture.get()).isSameAs(leaderSessionId);
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerServiceImplTest.java:363]: How about assertThatFuture?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerServiceImplTest.java [hunk 12]
```diff
@@ -385,18 +380,18 @@ public void closeService_stopRmAndLeaderElection() throws Exception {
...
-        assertFalse(leaderElection.isStopped());
+        assertThat(leaderElection.isStopped()).isFalse();
 
         // close service
         resourceManagerService.close();
 
         // should stop RM and leader election
-        assertTrue(terminateRmFuture.isDone());
-        assertTrue(leaderElection.isStopped());
+        assertThat(terminateRmFuture.isDone()).isTrue();
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerServiceImplTest.java:389]: How about assertThatFuture?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerServiceImplTest.java [hunk 13]
```diff
@@ -497,40 +492,31 @@ public void grantAndRevokeLeadership_verifyMetrics() throws Exception {
...
     }
 
     private static void assertNotComplete(CompletableFuture<?> future) throws Exception {
-        try {
-            future.get(50, TimeUnit.MILLISECONDS);
-            fail();
-        } catch (TimeoutException e) {
-            // expected
-        }
+        assertThatThrownBy(() -> future.get(50, TimeUnit.MILLISECONDS))
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerServiceImplTest.java:518]: How about assertThatFuture?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerTaskExecutorTest.java [hunk 14]
```diff
@@ -234,17 +225,12 @@ public void testDelayedRegisterTaskExecutor() throws Exception {
...
-                fail(
-                        "Should have failed because connection to taskmanager is delayed beyond timeout");
-            } catch (Exception e) {
-                final Throwable cause = ExceptionUtils.stripExecutionException(e);
-                assertThat(cause, instanceOf(TimeoutException.class));
-                assertThat(
-                        cause.getMessage(),
-                        containsString("ResourceManagerGateway.registerTaskExecutor"));
-            }
+            assertThatThrownBy(firstFuture::get)
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerTaskExecutorTest.java:228]: How about assertThatFuture?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerTaskExecutorTest.java [hunk 15]
```diff
@@ -336,31 +321,28 @@ private Collection<SlotStatus> createSlots(int numberSlots) {
...
                 registerTaskExecutor(wronglyFencedGateway, taskExecutorGateway.getAddress());
 
-        try {
-            unMatchedLeaderFuture.get();
-            fail(
-                    "Should have failed because we are using a wrongly fenced ResourceManagerGateway.");
-        } catch (ExecutionException e) {
-            assertTrue(ExceptionUtils.stripExecutionException(e) instanceof FencingTokenException);
-        }
+        assertThatThrownBy(unMatchedLeaderFuture::get)
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerTaskExecutorTest.java:330]: How about assertThatFuture?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerTaskExecutorTest.java [hunk 16]
```diff
@@ -336,31 +321,28 @@ private Collection<SlotStatus> createSlots(int numberSlots) {
...
-    public void testRegisterTaskExecutorFromInvalidAddress() throws Exception {
+    void testRegisterTaskExecutorFromInvalidAddress() throws Exception {
         // test throw exception when receive a registration from taskExecutor which takes invalid
         // address
         String invalidAddress = "/taskExecutor2";
 
         CompletableFuture<RegistrationResponse> invalidAddressFuture =
                 registerTaskExecutor(rmGateway, invalidAddress);
-        assertTrue(invalidAddressFuture.get() instanceof RegistrationResponse.Failure);
+        assertThat(invalidAddressFuture.get()).isInstanceOf(RegistrationResponse.Failure.class);
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerTaskExecutorTest.java:345]: How about assertThatFuture?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java [hunk 17]
```diff
@@ -175,7 +168,7 @@ public void testLessThanDeclareResource() throws Exception {
                                                             .requestNewWorker(WORKER_RESOURCE_SPEC))
                                     .get(TIMEOUT_SEC, TimeUnit.SECONDS);
 
-                            assertThat(requestCount.get(), is(2));
+                            assertThat(requestCount.get()).isEqualTo(2);
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java:171]:
```suggestion
                            assertThat(requestCount).hasValue(2);
```

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java [hunk 18]
```diff
@@ -193,15 +186,15 @@ public void testLessThanDeclareResource() throws Exception {
                             declareResourceFuture.get(TIMEOUT_SEC, TimeUnit.SECONDS);
 
                             // request new worker.
-                            assertThat(requestCount.get(), is(3));
+                            assertThat(requestCount.get()).isEqualTo(3);
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java:189]: get isn't needed.

Comment (1996fanrui) [reply flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java:189]: This class has lots of `AtomicInteger`, please update all of them, thanks~

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java [hunk 19]
```diff
@@ -471,22 +466,23 @@ public void testWorkerTerminatedBeforeRegister() throws Exception {
...
-                            assertThat(taskExecutorProcessSpec2, is(taskExecutorProcessSpec1));
+                            assertThat(taskExecutorProcessSpec2).isSameAs(taskExecutorProcessSpec1);
 
                             // second worker registered, verify registration succeed
                             CompletableFuture<RegistrationResponse> registerTaskExecutorFuture =
                                     registerTaskExecutor(tmResourceIds.get(1));
                             assertThat(
-                                    registerTaskExecutorFuture.get(TIMEOUT_SEC, TimeUnit.SECONDS),
-                                    instanceOf(RegistrationResponse.Success.class));
+                                            registerTaskExecutorFuture.get(
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java:475]: Is the get needed?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java [hunk 20]
```diff
@@ -531,19 +527,19 @@ public void testWorkerTerminatedAfterRegister() throws Exception {
...
-                                                            flinkConfig, WORKER_RESOURCE_SPEC)));
+                                                            flinkConfig, WORKER_RESOURCE_SPEC));
 
                             // first worker registered, verify registration succeed
                             CompletableFuture<RegistrationResponse> registerTaskExecutorFuture1 =
                                     registerTaskExecutor(tmResourceIds.get(0));
                             assertThat(
-                                    registerTaskExecutorFuture1.get(TIMEOUT_SEC, TimeUnit.SECONDS),
-                                    instanceOf(RegistrationResponse.Success.class));
+                                            registerTaskExecutorFuture1.get(
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java:540]: Is the get needed?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java [hunk 21]
```diff
@@ -557,22 +553,23 @@ public void testWorkerTerminatedAfterRegister() throws Exception {
...
-                            assertThat(taskExecutorProcessSpec2, is(taskExecutorProcessSpec1));
+                            assertThat(taskExecutorProcessSpec2).isSameAs(taskExecutorProcessSpec1);
 
                             // second worker registered, verify registration succeed
                             CompletableFuture<RegistrationResponse> registerTaskExecutorFuture2 =
                                     registerTaskExecutor(tmResourceIds.get(1));
                             assertThat(
-                                    registerTaskExecutorFuture2.get(TIMEOUT_SEC, TimeUnit.SECONDS),
-                                    instanceOf(RegistrationResponse.Success.class));
+                                            registerTaskExecutorFuture2.get(
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java:562]: Is the get needed?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java [hunk 22]
```diff
@@ -609,19 +606,19 @@ public void testWorkerTerminatedNoLongerRequired() throws Exception {
...
-                                                            flinkConfig, WORKER_RESOURCE_SPEC)));
+                                                            flinkConfig, WORKER_RESOURCE_SPEC));
 
                             // worker registered, verify registration succeed
                             CompletableFuture<RegistrationResponse> registerTaskExecutorFuture =
                                     registerTaskExecutor(tmResourceId);
                             assertThat(
-                                    registerTaskExecutorFuture.get(TIMEOUT_SEC, TimeUnit.SECONDS),
-                                    instanceOf(RegistrationResponse.Success.class));
+                                            registerTaskExecutorFuture.get(
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java:619]: Is the get needed?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java [hunk 23]
```diff
@@ -756,23 +753,23 @@ public void testStartWorkerIntervalOnWorkerTerminationExceedFailureRate() throws
...
+                            assertThat((t2 - t1))
+                                    .isGreaterThanOrEqualTo(
+                                            TESTING_START_WORKER_INTERVAL.toMilliseconds());
                             // second worker registered, verify registration succeed
                             CompletableFuture<RegistrationResponse> registerTaskExecutorFuture =
                                     registerTaskExecutor(tmResourceIds.get(1));
                             assertThat(
-                                    registerTaskExecutorFuture.get(TIMEOUT_SEC, TimeUnit.SECONDS),
-                                    instanceOf(RegistrationResponse.Success.class));
+                                            registerTaskExecutorFuture.get(
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java:763]: Is the get needed?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java [hunk 24]
```diff
@@ -837,26 +834,26 @@ public void testStartWorkerIntervalOnRequestWorkerFailure() throws Exception {
...
+                                            TESTING_START_WORKER_INTERVAL.toMilliseconds());
 
                             // second worker registered, verify registration succeed
                             resourceIdFutures.get(1).complete(tmResourceId);
                             CompletableFuture<RegistrationResponse> registerTaskExecutorFuture =
                                     registerTaskExecutor(tmResourceId);
                             assertThat(
-                                    registerTaskExecutorFuture.get(TIMEOUT_SEC, TimeUnit.SECONDS),
-                                    instanceOf(RegistrationResponse.Success.class));
+                                            registerTaskExecutorFuture.get(
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java:846]: Is the get needed?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java [hunk 25]
```diff
@@ -390,23 +385,24 @@ public void testStartNewWorkerFailedRequesting() throws Exception {
...
+                            assertThat(taskExecutorProcessSpec2).isSameAs(taskExecutorProcessSpec1);
 
                             // second request allocated, verify registration succeed
                             runInMainThread(() -> resourceIdFutures.get(1).complete(tmResourceId));
                             CompletableFuture<RegistrationResponse> registerTaskExecutorFuture =
                                     registerTaskExecutor(tmResourceId);
                             assertThat(
-                                    registerTaskExecutorFuture.get(TIMEOUT_SEC, TimeUnit.SECONDS),
-                                    instanceOf(RegistrationResponse.Success.class));
+                                            registerTaskExecutorFuture.get(
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java:395]: Is the get needed?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java [hunk 26]
```diff
@@ -940,16 +939,15 @@ public void testWorkerRegistrationTimeout() throws Exception {
                                                     .requestNewWorker(WORKER_RESOURCE_SPEC));
 
                             // verify worker is released due to not registered in time
-                            assertThat(
-                                    releaseResourceFuture.get(TIMEOUT_SEC, TimeUnit.SECONDS),
-                                    is(tmResourceId));
+                            assertThat(releaseResourceFuture.get(TIMEOUT_SEC, TimeUnit.SECONDS))
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java:942]: Is the get needed?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java [hunk 27]
```diff
@@ -871,32 +868,34 @@ public void testRecoverWorkerFromPreviousAttempt() throws Exception {
...
         new Context() {
             {
                 runTest(
                         () -> {
                             CompletableFuture<RegistrationResponse> registerTaskExecutorFuture =
                                     registerTaskExecutor(ResourceID.generate());
                             assertThat(
-                                    registerTaskExecutorFuture.get(TIMEOUT_SEC, TimeUnit.SECONDS),
-                                    instanceOf(RegistrationResponse.Rejection.class));
+                                            registerTaskExecutorFuture.get(
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java:889]: Is the get needed?

File: flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java [hunk 28]
```diff
@@ -871,32 +868,34 @@ public void testRecoverWorkerFromPreviousAttempt() throws Exception {
                             CompletableFuture<RegistrationResponse> registerTaskExecutorFuture =
                                     registerTaskExecutor(tmResourceId);
                             assertThat(
-                                    registerTaskExecutorFuture.get(TIMEOUT_SEC, TimeUnit.SECONDS),
-                                    instanceOf(RegistrationResponse.Success.class));
+                                            registerTaskExecutorFuture.get(
```
Comment (1996fanrui) [inline flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManagerTest.java:871]: Is the get needed?

**Review by 1996fanrui — CHANGES_REQUESTED** [review]
Thanks @RocMarshal for the contribution!

I left some comments, please take a look in your free time, thanks~

**Comment by RocMarshal** [comment]
Hi， Thanks so much for @1996fanrui your review and patience.

I updated it based on your comments.
And I triggered about 4 times ci-flink-bot by modifying commit(--amend). Its succeeded final.

PTAL in your free time~  :)

**Review by 1996fanrui — APPROVED** [review]
Thanks @RocMarshal for the hard work!

LGTM

**Pull request merged**